webapp/.eslintrc.json

```diff
@@ -209,7 +209,7 @@
 		"react/jsx-uses-react": 2,
 		"react/jsx-uses-vars": 2,
 		"react/no-danger": 0,
-		"react/no-did-mount-set-state": 2,
+		"react/no-did-mount-set-state": [2, "allow-in-func"],
```

Comment:
Can you revert this rule? It's meant to discourage the use of callbacks like this and a more React friendly coding style.